src/librustc/hir/intravisit.rs

```diff
@@ -298,6 +298,9 @@ pub trait Visitor<'v> : Sized {
     fn visit_fn(&mut self, fk: FnKind<'v>, fd: &'v FnDecl, b: BodyId, s: Span, id: NodeId) {
         walk_fn(self, fk, fd, b, s, id)
     }
+    fn visit_use(&mut self, path: &'v Path, id: NodeId, hir_id: HirId) {
+        walk_use(self, path, id, hir_id)
+    }
```

Comment:
This and `walk_use` are unused.

Reply:
They are used because it means the visitor walks down into the paths and path segments in a `use` statement.

Reply:
`visit_use` is used in `walk_item`